allen-cell-animated/cell-catalog
# add stem cell characteristics data

Problem
=======
What is the problem this work solves, including
closes #55 

Solution
========
What I/we did to solve this problem
added fields for stem cell characteristics data to the config file
with @meganrm @tyler-foster @interim17 

## Type of change
Please delete options that are not relevant.

* New feature (non-breaking change which adds functionality)


Steps to Verify:
----------------
1. `npm run`
2. `npx netlify-cms-proxy-server`
3. check http://localhost:8000/admin/#/collections/disease-cell-line/entries/AICS-104/index

Screenshots (optional):
-----------------------
Show-n-tell images/animations here
<img width="671" alt="Screenshot 2024-09-12 at 2 28 50 PM" src="https://github.com/user-attachments/assets/4c642271-4345-4eec-afbc-6919fa60b5c9">
